packages/babel-plugin-ember-test-metadata/get-normalized-file-path.js

```diff
@@ -1,36 +1,32 @@
-const {
-  _getRelativePathForClassic,
-  _getRelativePathForClassicInRepo,
-  _getRelativePathForEmbroider,
-  _getRelativePathForEmbroiderInRepo,
-} = require('./get-relative-paths');
-
-const path = require('path');
-
 /**
  * Get a normalized file path, based on whether the app build is classic or with Embroider
  * @param {object} opts Babel state.file.opts which include root and filename props
```

Comment:
Can we use jsdoc to list out the actual Babel `state.file.opts`? This will make it easier for readers to understand where these params are originating from, and give some clue into their purpose.